ruby_on_rails/app_initialisation.md

```diff
@@ -72,7 +71,22 @@ They are always idempotent (runnable multiple times).
   ```
 
   Also change `system!('bundle install')` to `system!('bundle install --jobs=3 --retry=3')`
-  and uncomment the installation of yarn dependencies with `system('bin/yarn')`
+
+#### If you are using a node-based JS bundler using `jsbundling-rails`
+
+* Add the following to `bin/setup` to allow intalling all dependencies in one go
+
+  ```ruby
+  system! 'yarn install'
```

Comment:
Shouldn't it actually be `bin/yarn install --check-files`? That's at least the recommended command in the error message of Rails when you're dependencies are out of date